packages/grafana-ui/src/slate-plugins/slate-prism/index.ts

```diff
@@ -15,7 +27,23 @@ export function SlatePrism(optsParam: OptionsFormat = {}): Plugin {
       if (!opts.onlyIn(node)) {
         return next();
       }
-      return decorateNode(opts, Block.create(node as Block));
+
+      const block = Block.create(node as Block);
```

Comment:
Moved some logic out of `decorateNode` to here so I could expose the tokens generated by prism via the slate `Editor` object (so that we don't need to parse CSS classes to find the tokens).
I should break this logic out into a separate function to make things a bit tidier though